flocker/control/test/test_protocol.py

```diff
@@ -574,6 +575,55 @@ def test_nodestate_notifies_all_connected(self):
             ),
         )
 
+    def test_nodestate_coalesces_multiple_quick(self):
+        """
+        Multiple ``NodeStateCommands`` are coalesced into a single state update
+        broadcast to all the nodes.
+        """
+        self.control_amp_service.configuration_service.save(TEST_DEPLOYMENT)
+
+        agents = [FakeAgent(), FakeAgent()]
+        clients = list(AgentAMP(Clock(), agent) for agent in agents)
+        servers = list(LoopbackAMPClient(client.locator) for client in clients)
+
+        for server in servers:
+            self.control_amp_service.connected(server)
+
+        initial_update_counts = list(
+            agent.cluster_updated_count for agent in agents)
```

Comment:
I actually think this might be filled with a bunch of zeros, or maybe [1, 2, 3, 4, 5...] from the initial state updates. The real reason for taking these values is just because I only want to measure the number of times things are sent between the node state updates.